src/coredump/coredumpctl.c

```diff
@@ -515,9 +552,13 @@ static int print_list(FILE* file, sd_journal *j, Table *t) {
 
         normal_coredump = streq_ptr(mid, SD_MESSAGE_COREDUMP_STR);
 
-        if (filename)
+        if (filename) {
+                r = resolve_filename(filename, arg_root, &filename);
+                if (r < 0)
+                        return log_error_errno(r, "Failed to resolve coredump: %m");
```

Comment:
We generally don't log failures when calling a function that already logs on its own unless we can add extra valuable information. Since we can't really do that here, it's fine to just propagate the error from resolve_filename() without logging again. Same for the other callsites of resolve_filename().